flink-core/src/main/java/org/apache/flink/configuration/CoreOptions.java

```diff
@@ -210,7 +210,7 @@
 
 	/**
 	 * The config parameter defining the directories for temporary files, separated by
-	 * ",", "|", or the system's {@link java.io.File#pathSeparator}.
+	 * "," or the system's {@link java.io.File#pathSeparator}.
```

Comment:
Update also the description in `TMP_DIRS.withDescription` few lines below.

Reply:
Have fixed.